uint8[]   sensorReadID
uint16[]  sensorStartAddress
uint16[]  sensorReadLength
int32[]   sensorData
uint8     sensorCount
uint8     dataLength